src/jdk.httpserver/share/classes/module-info.java

```diff
@@ -37,6 +37,58 @@
  * <p>The {@link com.sun.net.httpserver.spi} package specifies a Service Provider
  * Interface (SPI) for locating HTTP server implementations based on the
  * {@code com.sun.net.httpserver} API.
+ * <p>
+ * <b id="httpserverprops">System properties used by the HTTP server API</b>
+ * <p>
+ * The following is a list of JDK specific system properties used by the HTTP
+ * server implementation.
```

Comment:
Hello Michael, given that the HttpServer is based on a SPI and the `sun.net.httpserver.HttpServerImpl`  (and its Https equivalent) are just default implementations, should either of these 2 sentences note that these system properties are only applicable for the default HttpServer implementation? Something like:

>
><b id="httpserverprops">System properties used by the default HTTP server</b>
The following is a list of JDK specific system properties used by the default HTTP server implementation.

Reply:
Maybe, I should include the property that is used to locate the impl as first in the list and say the remaining properties apply to the default impl in the JDK?

Reply:
No. That property should probably be deprecated since there is a more modern (ServiceLoader) alternative implemented.

Reply:
Good point. But if we wanted to document this property here, I think it would deserve a separate paragraph, before we start talking about properties, that talks about the SPI and explain how to provide an implementation - where the canonical way (providing an implementation of the provider that the ServiceLoader loads) would be documented first. As you say, that property is already documented in the SPI, and it's probably better to only document it at the place where we explain  how the SPI is loaded.